inception-project/inception
# #1211 - Only predict on all documents when in Active Learning Mode

**What's in the PR**
- Give the current document to the recommender tasks
- Check the current document when predicting
- Make sure that AL gets all predictions
- Requeue duplicate tasks to the end

**How to test manually**
- Set log level to debug and check that normally, only the current document is used for predictions
- Check that for Active Learning, all documents are used for prediction
- Check that after AL is done, it is single doc again
- Check that after AL switches documents, it is all docs again
- Check that predictions are generated after switching docs without making annotations

**Automatic testing**

**Documentation**
